packages/manager/modules/cloud-connect/src/overview/remove-pop-configuration/remove-pop-configuration.controller.js

```diff
@@ -0,0 +1,63 @@
+import get from 'lodash/get';
+
+export default class RemovePopConfigurationCtrl {
+  /* @ngInject */
+  constructor($translate, cloudConnectService) {
+    this.$translate = $translate;
+    this.cloudConnectService = cloudConnectService;
+  }
+
+  $onInit() {
+    this.isLoading = false;
+  }
+
+  removePopConfigure() {
+    this.cloudConnectService.trackClick(
+      'cloud-connect::overview::remove-pop::confirm',
+    );
+    this.isLoading = true;
+    this.cloudConnectService
```

Comment:
Maybe you should use `return`, in case if you need to call this method, and control the asynchronous flow